test/test_recordings_filter.py

```diff
@@ -59,3 +59,27 @@ def test_latitude_longitude_filter(self, helper):
         print("  Recordings from a normal query shoudl also be filtered", end="")
         rec = bob.query_recordings(filterOptions=json.dumps({"latLongPrec": 10}))[0]
         assert rec["location"]["coordinates"] == [20.00025, 20.00025]
+
+    def test_recordfing_query_fields(self, helper):
+        larry = helper.given_new_user(self, "larry_confident")
+        larry_group = helper.make_unique_group_name(self, "larrys_group")
+        larry.create_group(larry_group)
+        larry_device = helper.given_new_device(self, "larrys_device", larry_group)
+        recording = larry_device.upload_recording()
+        track = larry.can_add_track_to_recording(recording)
+        larry.can_tag_track(track, automatic=False, what="possum")
+        recording.is_tagged_as(what="foo", detail="bar").by(larry)
+
+        response = larry.query_recordings(return_json=True)
+        for r in response["rows"]:
+            tags = r.get("Tags")
+            assert tags is not None
```

Comment:
Instead of using get() and then checking for None you could just `tags = r["Tags"]` which will fail with a explantory `KeyError` if it fails. Same for the similar cases below.